app/controllers/idv/in_person/ready_to_verify_controller.rb

```diff
@@ -10,7 +10,12 @@ class ReadyToVerifyController < ApplicationController
       before_action :confirm_in_person_session
 
       def show
-        analytics.idv_in_person_ready_to_verify_visit
+        bucket = session[:in_person_cta_variant]
```

Comment:
The `Analytics` class in `analytics.rb` has direct access to the session, so you could set the value in the `analytics_hash` there and avoid the need to update all of the touchpoints.

Reply:
I believe what you're describing is the approach we took in [PR#7750](https://github.com/18F/identity-idp/pull/7750) which raised some concerns about passing variant data in scenarios it wasn't needed. The goal of this rewrite was to include variant data only in those log lines we specifically would be using for metrics, since there are only 6.

Reply:
I think [the approach](https://github.com/18F/identity-idp/pull/7750/files#r1096306129) mentioned by @aduth also makes sense, but I'm necessarily opposed to updating the individual touchpoints if that seems more practical.

Reply:
Is it possible to re-calculate this from the same flow session UUID data? or is that deleted by the time we get here?

Reply:
We could recalculate it, but since the session ID changes after logout/login we would need to either exclude data from subsequent logins or persist it more permanently.

Reply:
I'm inclined to forego any changes with this that are not strictly necessary. There's upcoming work to change how the bucket is hashed, which might change the conversation.